### PR TITLE
Updated index.md

### PR DESCRIPTION
**Pull request by Vipulagarwal-2000**
fixed the sentence structure from "not learning a new tool" to "learning a new tool" which sound correct.

### Description
This is the original sentence.

**With this knowledge, you can focus on your contributions and not learn a new tool at the same time.**

This is the modified sentence

**With this knowledge, you can focus on your contributions and learn a new tool at the same time.**

### Motivation
This is my first open-source contribution. I am excited about doing this. Second, this is a simple typo, but it will make the sentence clear.

### Additional details

Here is the page link:
[page link](https://developer.mozilla.org/en-US/docs/MDN/Community/Contributing/Getting_started)

### CODE REVIEW

File: files/en-us/mdn/community/contributing/getting_started/index.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ We have created a [contributors task board](https://github.com/orgs/mdn/projects
 
 ### Getting ready to contribute
 
-To contribute, you will need a GitHub account. If you do not already have one, go ahead and [sign up](https://github.com/signup) for an account before continuing. If you are new to GitHub, we encourage you to take the following free, self-paced courses and reading material offered by GitHub. With this knowledge, you can focus on your contributions and not learn a new tool at the same time.
+To contribute, you will need a GitHub account. If you do not already have one, go ahead and [sign up](https://github.com/signup) for an account before continuing. If you are new to GitHub, we encourage you to take the following free, self-paced courses and reading material offered by GitHub. With this knowledge, you can focus on your contributions and learn a new tool at the same time.
 
 > NOTE: Do not feel overwhelmed or like you have to read through and complete _all_ of the course work. With the knowledge gained from the "Introduction to GitHub" course, you will be well on your way.
```
Comment (bsmth) [inline files/en-us/mdn/community/contributing/getting_started/index.md:28]:
```suggestion
> [!NOTE]
> Do not feel overwhelmed or like you have to read through and complete _all_ of the course work. With the knowledge gained from the "Introduction to GitHub" course, you will be well on your way.
```

File: files/en-us/mdn/community/contributing/getting_started/index.md [hunk 2]
```diff
@@ -23,9 +23,10 @@ We have created a [contributors task board](https://github.com/orgs/mdn/projects
 
 ### Getting ready to contribute
 
-To contribute, you will need a GitHub account. If you do not already have one, go ahead and [sign up](https://github.com/signup) for an account before continuing. If you are new to GitHub, we encourage you to take the following free, self-paced courses and reading material offered by GitHub. With this knowledge, you can focus on your contributions and not learn a new tool at the same time.
+To contribute, you will need a GitHub account. If you do not already have one, go ahead and [sign up](https://github.com/signup) for an account before continuing. If you are new to GitHub, we encourage you to take the following free, self-paced courses and reading material offered by GitHub. With this knowledge, you can focus on your contributions and learn a new tool at the same time.
```
Comment (bsmth) [inline files/en-us/mdn/community/contributing/getting_started/index.md:26]: I think this is what the original means:

```suggestion
To contribute, you will need a GitHub account. If you do not already have one, go ahead and [sign up](https://github.com/signup) for an account before continuing. If you are new to GitHub, we encourage you to take the following free, self-paced courses and reading material offered by GitHub. With this knowledge, you can focus on your contributions without the burden of learning a new tool at the same time.
```

Comment (Vipulagarwal-2000) [reply files/en-us/mdn/community/contributing/getting_started/index.md]: yeah, that make more sense.

**Review by Vipulagarwal-2000 — COMMENTED** [review]
yeah now it look better and perfect.

**Review by bsmth — APPROVED** [review]
Super, thanks a lot!

**Pull request merged**